src/main/java/com/puppycrawl/tools/checkstyle/api/TokenTypes.java

```diff
@@ -2215,6 +2215,21 @@
      * The {@code default} keyword.  This element has no
      * children.
      *
+     * <p>For example:</p>
+     * <pre>
+     * default int x;
```

Comment:
it will no compile, as far as I know `default`  is used in three cases
- switch statement
- annotation
- interface method implementation

try to use compilable example

Reply:
so I have to show for all threee cases in one program?

Reply:
no, just one, any of them